app/src/main/java/com/amaze/filemanager/asynchronous/services/ServiceWatcherProgressAbstract.java

```diff
@@ -31,9 +32,25 @@
     public ProgressHandler progressHandler;
     public Context context;
     // list of data packages, to initiate chart in process viewer fragment
-    public ArrayList<CopyDataParcelable> dataPackages = new ArrayList<>();
+    public ArrayList<CopyDataParcelable> dataPackages;
     public ProgressListener progressListener;
 
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        initVariables();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+
+        initVariables();
```

Comment:
Twice for good measure?

Reply:
Ahh, my bad. Forgot to remove one in onCreate